test/tests/parallel/set1/spam_virus/postgrey_enabled.bats

```diff
@@ -60,6 +60,10 @@ function teardown_file() { _default_teardown ; }
     'action=greylist' \
     'reason=new' \
     'client_address=127.0.0.1/32, sender=user@external.tld, recipient=user1@localhost.localdomain'
+
+  _repeat_until_success_or_timeout 10 _run_in_container grep \
+    'Recipient address rejected: Delayed by Postgrey' \
+    /var/log/mail/mail.log
```

Comment:
This addition looks like it was shifted up during your refactor for adding swaks.

The call directly above is failing at line 60. This is because the client address value needs to be `127.0.0.1` instead of `127.0.0.1/32` like the other related changes to this file below.

Reply:
Done in 68a67c95e2a5b7ecc588a4c2acdd25f64d3e17ff